tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -27,90 +28,57 @@
 class test_TangoSimGenDeviceIntegration(ClassCleanupUnittestMixin, unittest.TestCase):
```

Comment:
The name of the base class should not start with `test_` otherwise it raise error when nosetests tries to run it.